tests/c/indirect_call.newcg.c

```diff
@@ -0,0 +1,64 @@
+// ignore-if: test $YK_JIT_COMPILER != "yk" -o "$YKB_TRACER" = "swt"
+// Run-time:
+//   env-var: YKD_LOG_IR=-:aot,jit-pre-opt
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_LOG_JITSTATE=-
+//   stderr:
+//     jitstate: start-tracing
+//     foo 7
+//     jitstate: stop-tracing
+//     --- Begin aot ---
```

Comment:
Is the AOT match checking anything useful here?

Reply:
No. It was just copied over from the other test. I can remove it.

Reply:
[9883a3a](https://github.com/ykjit/yk/pull/1141/commits/9883a3a605edc40a905df1fed200a5c473bc3fb6)